test/test_edgerc

```diff
@@ -1,11 +1,12 @@
 [default]
 host = example.luna.akamaiapis.net
 client_token = clientToken
-client_secret = clientSecret
+client_secret = "client;Secret" ; there is a semi-colon in the client secret
 access_token = accessToken
 max-body = 131072
+; line comment
 
-[section]
+[section] ;this is a section
 host = sectionexample.luna.akamaiapis.net
 client_token = sectionClientToken
 client_secret = sectionClientSecret
```

Comment:
I would add a new section and ensure that it parses:

```ini
; comment
[comment-test] ; comment
;comment
; comment
host = "sectionexample.luna.akamaiapis.net" ; comment
client_token = "client token" ;comment
client_secret = "client;secret" ; comment
access_token = "access; token" ;comment
max-body = 131072
; comment
```

Reply:
Done. I added the section (and the tests) and removed comments from other sections